settings/src/components/backup-codes.js

```diff
@@ -241,8 +236,10 @@ function Manage( { setRegenerating } ) {
 				) }
 			</div>
 
-			<Button isSecondary onClick={ () => setRegenerating( true ) }>
-				Generate new backup codes
+			<Button isSecondary onClick={ () => setGenerating( true ) }>
+				{ backupCodesEnabled
+					? 'Regenerate and save backup codes'
+					: 'Generate and save backup codes' }
```

Comment:
I understand why we want to add "Generate and **save backup codes**" because that's what technically happens but I don't know that first time users will understand what save means here. I could be wrong.